DevExtreme CLI: add React Next.js info (#7295)
DevExpress/devextreme-documentation: concepts/50 React Components/50 Application Template/40 Configure Themes/40 Apply Theme Variables to Custom Elements.md

> Theme variables are defined in `src\variables.scss`. Apply the variables to custom elements, so that the elements have a uniform appearance with the rest of the application.

Reviewer: ```suggestion
Theme variables are defined in `src\variables.scss`. Apply the variables to custom elements, so that element appearance is consistent with the rest of the application.
```
